verify/module.py

```diff
@@ -1634,10 +1662,9 @@ def _check_inbox_for_errors(self):
                 m
                 for m in mailbox.fetch(
```

Comment:
Change name of the variable to "message" or something more descriptive.

Reply:
I think this one should stay as it's in a generator expression. However with the for cycle you are right.